tests/sentry/api/endpoints/relocations/test_public_key.py

```diff
@@ -44,6 +45,43 @@ def test_good_superuser_when_feature_enabled(
         assert bytes(response.data["public_key"], "utf-8") == self.pub_key_pem
         assert fake_kms_client.get_public_key.call_count == 1
 
+    def test_good_superuser_when_feature_disabled(
+        self, fake_kms_client: FakeKeyManagementServiceClient
+    ):
+        superuser = self.create_user("superuser", is_superuser=True, is_active=True)
+        self.login_as(user=superuser, superuser=True)
+        self.mock_kms_client(fake_kms_client)
+        response = self.get_success_response(status_code=200)
+
+        assert response.status_code == 200
+        assert bytes(response.data["public_key"], "utf-8") == self.pub_key_pem
+        assert fake_kms_client.get_public_key.call_count == 1
+
+    @override_options({"relocation.enabled": True})
+    @with_feature("auth:enterprise-staff-cookie")
```

Comment:
Sorry I'm actually right in the middle of switching over from using a staff feature flag to an option for the rollout: https://github.com/getsentry/team-core-product-foundations/issues/224

You can use the previous helpers as expected, but to test for staff now you use the `staff.ga-rollout` option like:
```suggestion
    @override_options({"relocation.enabled": True, "staff.ga-rollout": True})
```

Reply:
Fixed here and elsewhere in the file.